doc-content/jbpm-docs/src/main/asciidoc/Integration/integration-maven-dependency-con.adoc

```diff
@@ -0,0 +1,177 @@
+[id='integration-maven-dependency-con_{context}']
+= Dependency management with Maven
+
+When you build projects that embed the {PROCESS_ENGINE}, use Apache Maven to configure all dependencies required by the {PROCESS_ENGINE}.
+
+The {PROCESS_ENGINE} provides a set of BOMs (Bill Of Material) to simplify declaring artifact dependencies.
```

Comment:
Since it's the plural "BOMs", I guess should say "(Bills of Material)", as in "set of Bills of Material".